Jenkinsfile

```diff
@@ -19,51 +19,48 @@ pipeline {
         grantIPAccess()
       }
     }
+
     stage('Validate') {
-      parallel {
-        stage('Changelog') {
```

Comment:
Do we want to keep the `stage('Changelog') {` ?  Or maybe change the name of the parent stage `Validate Changelog` (Line 23)? i.e. would be nice to see what we're validating in the Jenkins pipeline?

Reply:
Changed to `Validate Changelog`